### DISCUSSION TITLE
System requirements

### DISCUSSION THREAD

**Discussion by flockelhahn** [open-mmlab/OpenLidarPerceptron#1245] [Q&A]
Hello,

I am fairly new to the topic of object detection (masters thesis) and wanted to ask what the System requirements are, to properly work with tools like OpenPCDet. Maybe you have some info, about what works good/bad, in case of GPU/CPU.

Thanks in advance!

**Comment by jihanyang** [comment]
Linux OS is recommanded with any GPU can support pytorch1.1 ~ latest pytorch version. Notice that CUDA > 10.2 is recommanded for spconv 2.x.

**Reply by Tejaswini011** [reply]
Does it supports in VM

**Reply by auggie246** [reply]
> Linux OS is recommanded with any GPU can support pytorch1.1 ~ latest pytorch version. Notice that CUDA > 10.2 is recommanded for spconv 2.x.

What about VRAM requirements? Stated in the readme that 8x 1080 Ti was used but for CaDNN model is all the vram used?

**Comment by A-Janj** [comment]
I am having issues running this repo in google colab. Can anyone guide me?
